aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/internal/TimeUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.xray.internal;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+public final class TimeUtils {
+    /**
+     * @return the current epoch second
+     */
+    public static long currentEpochSecond() {
+        return MILLISECONDS.toSeconds(System.currentTimeMillis());
+    }
+
+    private TimeUtils() {
+        throw new AssertionError("no instances, please");
+    }
```

Comment:
```suggestion
    private TimeUtils() {}
```

Think we don't need to add the extra noise